### PR TITLE
Adding toml resource

### PR DESCRIPTION
**Pull request by nsdavidson**
This adds a `toml` resource that inherits from the json resource and
behaves the same way as the JSON and YAML resources.

Signed-off-by: Nolan Davidson <ndavidson@chef.io>

### CODE REVIEW

File: lib/resources/toml.rb [hunk 1]
```diff
@@ -0,0 +1,33 @@
+# encoding: utf-8
+# author: Nolan Davidson
+
+require 'toml'
+
+# Parses a TOML document
+# Usage
+# describe toml('default.toml') do
```
Comment (chris-rock) [inline lib/resources/toml.rb]: We do not need the comments, since the example is inline as part of the `example`

**Review by chris-rock — COMMENTED** [review]
Great addition @nsdavidson Lets just remove the additional comments and we are good to go :-)

**Comment by nsdavidson** [comment]
@chris-rock Thanks!  Comments removed.

**Review by adamleff — APPROVED** [review]
Nicely done, @nsdavidson!

**Pull request merged**